Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -44,6 +44,8 @@ protected UITableViewRowAnimation ReloadSectionsAnimation
 			set { _dataSource.ReloadSectionsAnimation = value; }
 		}
 
+		protected virtual bool ActionCellIconInsteadOfText { get; }
```

Comment:
Not sure about the naming  `ContextActionShouldUseIcon` ?!

Reply:
You can name it anything you want. It just shows where changes are required for a minimal change and without changing the behavior on any existing app.